src/index.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express'
+import passport from 'passport'
+//import * as babel from '@babel/core';
+let app=express()
+require('dotenv').config()
```

Comment:
Hi @munyanezaarmel,

This line of code is not an ES6 line of code, please make adjustment